__tests__/app.test.js

```diff
@@ -88,4 +88,33 @@ describe('GET /api/articles/:article_id', () => {
             expect(body.msg).toBe('not found')
         })
     })
+})
+describe('GET: /api/articles', () => {
+    test('GET 200: should respond with an array of article objects, each with the correct properties and data types for each property', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            const articles = body
+            articles.forEach((article) => {
+                expect(typeof article.article_id).toBe('number')
+                expect(typeof article.author).toBe('string')
+                expect(typeof article.title).toBe('string')
+                expect(typeof article.topic).toBe('string')
+                expect(typeof article.created_at).toBe('string')
+                expect(typeof article.votes).toBe('number')
```

Comment:
This is an excellent test, and you're even making sure that the body _isn't_ on the objects which is brilliant 🏅 
One thing I notice is that on line `98` you aren't destructuring which would look like `const { articles } = body;` this likely means the array isn't being send in an object from the controller